site/content/3.10/arangograph/organizations/_index.md

```diff
@@ -35,18 +35,29 @@ member of one _Free-to-try_ tier organization at a time.
 With the ArangoGraph Insights Platform, your organization can choose one of the
 following packages.
 
-### Free Trial
+### Perpetual Free Trial
 
-ArangoGraph comes with a free-to-try tier that lets you test ArangoGraph for
-free for 14 days. You can get started quickly, without needing to enter a
-credit card.
+ArangoGraph comes with a perpetual free tier that lets you access the platform
+without incurring any costs indefinitely. You can get started quickly, without
+needing to enter a credit card.
 
-The free trial gives you access to:
-- One small deployment (4GB) in a region of your choice for 14 days
+The perpetual free tier is ideal for anyone who wants to explore ArangoGraph
+and run smaller-scale development projects, experimentation, and prototyping.
+
+The perpetual free trial gives you access to:
+- One small deployment (4GB) limited to a few selected regions (AWS Ohio and AWS Ireland)
```

Comment:
Is this Single Server, or can it also be OneShard / Sharded?

```suggestion
- One small deployment (4 GB) limited to a few selected regions (AWS Ohio and AWS Ireland)
```

Maybe specify that 4 GB is the memory size? Because there is also "40 GB", which is the storage capacity associated with a 4 GB RAM deployment.

In case this can be a cluster, the 4 GB memory is per node!